client/app/components/form/UserOperatorMultiStepForm.tsx

```diff
@@ -101,12 +100,27 @@ export default function UserOperatorMultiStepForm({
   const isFormStatusDisabled =
     formData?.status === Status.PENDING || formData?.status === Status.APPROVED;
 
+  const isReviewWithRequestChanges =
+    isCasInternal && formData.is_new && formSection === 1;
+  const isReviewWithoutRequestChanges = isCasInternal && formSection === 2;
+  const isNewOperatorMessage =
+    !isCasInternal && formData.is_new && formSection === 1;
+
   const operatorRoute = isCasInternal ? "operators" : "select-operator";
   // If the user is an approved cas internal user or if no operator exists show the entire multistep form
   if (isCasInternal || !userOperatorId || formSection) {
     return (
       <>
-        {isCasInternal && formData.is_new && formSection === 0 && (
```

Comment:
This wasn't displaying correctly as it was using index counting (variable `formSectionIndex`) and not the actual page count `formSection` ie page 1. This can be verified by logging in as internal user and viewing `New Operator 3 Legal Name` User Operator.

<img width="1454" alt="Screenshot 2024-01-25 at 7 45 40 AM" src="https://github.com/bcgov/cas-registration/assets/14259474/cae73332-8768-472e-b8d0-e9d36671384f">